tests/test_core.py

```diff
@@ -2169,6 +2169,11 @@ def test_memorygrowth_3_force_fail_reallocBuffer(self):
     self.emcc_args += ['-Wno-almost-asm', '-s', 'ALLOW_MEMORY_GROWTH=1', '-s', 'TEST_MEMORY_GROWTH_FAILS=1']
     self.do_run_in_out_file_test('tests', 'core', 'test_memorygrowth_3')
 
+  def test_memorygrowth_new_error(self):
+    # test that C++ new properly errors if we fail to malloc
+    self.emcc_args += ['-Wno-almost-asm', '-s', 'ALLOW_MEMORY_GROWTH=1', '-s', 'MAXIMUM_MEMORY=18MB']
+    self.do_run_in_out_file_test('tests', 'core', 'test_memorygrowth_new_error')
```

Comment:
Does doesn't seem related to memory growth.   Why not just not set any settings and leave the default 16Mb memory.  The result should be the same.

Then we can just call this test something like: `test_aborting_new`?    (The name "new error" makes it sounds to me like new is an adjective).

Reply:
I'll add a comment - this is actually related to memory growth. When growth is disabled, we abort() in another code path (the code to grow memory looks very different in the two cases).